styles.css

```diff
@@ -1,22 +1,54 @@
 * {
-  font-family: 'Arial';
+  font-family: "Arial";
+}
+
+body {
+  text-align: center;
+}
+
+#play-button {
+  padding-left: 1rem;
+  margin-bottom: 1rem;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  padding-right: 1rem;
+  border-radius: 0.1rem;
+  border: none;
 }
 
 #board {
   display: grid;
-  grid-template-rows: 1fr 1fr;
-  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 50% 50%;
+  grid-template-columns: 50% 50%;
+  width: fit-content;
+  text-align: justify;
+  margin: auto;
+  margin-bottom: 1rem;
+}
+
+.score {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  width: 40%;
   margin: auto;
-  width: 80%;
 }
 
 .square {
-  padding: 2rem;
-  border: 1px solid blue;
+  border: dotted;
+  border-color: blue;
+  width: 4rem;
+  padding: 4rem;
   background-color: rgba(173, 216, 230, 0.5);
   cursor: pointer;
 }
 
+footer {
+  position: absolute;
+  bottom: 0;
+  width: 99%;
+  margin: auto;
+}
+
 .square:active {
```

Comment:
I initially added this as a test, but we should probably remove it now so that it doesn't seem like the user can click when the click events are actually disabled such that clicking doesn't actually do anything
```suggestion

```